lib/requestIDs.js

```diff
@@ -0,0 +1,23 @@
+import uuid from "uuid/v1"
+
+export function headers({ requestID, traceId, parentSpanId }) {
+  return {
+    "X-Request-Id": requestID,
+    "x-datadog-trace-id": traceId,
+    "x-datadog-parent-id": parentSpanId,
```

Comment:
@saolsen I noticed that these last two are not strings. I’m sure that all entries in `response.headers` get coerced to strings by Express before the response is sent to the client, but it was slightly unexpected to see these objects in there. Do you think we can add `toString()` calls here?